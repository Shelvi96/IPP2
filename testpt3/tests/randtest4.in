NEW ID

7;95;3>14
44>4
0>6;286932:
596>459
956:19253:>74
77714;>210375;0
1>;;;092944;
9>74600
866095>8901;31:4:
16946:312>2662
421981603>344;6010
977001>6;;8;3
40;6;39461>6;1;;
8:21;32>726655;:
5>;659;:3:
6993>;
77:424:>2
30701:>:85033725
5181>738
8;2:6522>744502:
95:2277>0;:3
8;7215;3;>:4
888689364>5;153
93>021271
;2838434>0
9::;4>0144
7>39
9926>4231:
942>6155976:
:1;5>06598;956
:43>1977:04;;2
;432:;284>5352124
53;62:572>6
:911;;46>8
547;5>336:61
8:836605>1
35395>;277
5880>:74;72
:86:52>4597726
4699;>9
97:5;6>:4
87>2;508472
9:877>18;23972
6585>36
6382;7824>:35:819:8
9;:>8:115;1;71
68185778>595
7>0942
0>5695;8015
16765279;>693090849
82>6;:6
;0:565>4;:522
;680:4479>2871;112
3708;8;9>1050:140:9
;2:01;;24>8710708691
17;30442>0272
0516262>;6
36;1>;
72:7:2:>6420
25;979399>60895
803;2>94
525;>36723
3:84>291;2
0653>444996290
30;:72>7;598:0
;09439086>9436501533
:439105>07:1878:78
96040>56209::517
54560>;07;:2823
78605>75298;376
03643>8:975518
88515:0349>30:192
7:4573>2
;>0:
97565>912097;:85
3448;;:131>3
0475;4>06513
4>899;15
08:>;02232;2
:7:>533
7>93967237
;94:165514>1663
3;4>635:43
;7>::129
1602>;
86:41909>8
:49>3435
88>;92
30>:8904:
57:5109;7>7::26:
:>012440
5598;6793>9008
;:028:46:5>098526:
29>1:42;827
2>68580469
774>1:284;9
15801>;;505552
10>97
2;7>139
4>99312642;4
42>232511
756;:987>34
7:412:5>:113
4909;;>854
6>5;8:5
820205>:;22
7>;4221:85
61269438>5976208
3:022368>;;548062:7
143;125735>6858:86:
69729;04>03246199:8
41:06:268>06279
:1>;1:1957:
5>218417231:
1>:0
92751>33:029
;8014>63::5
:::834>53278:2724
53858>234451533
;6;807>;351
;9244>25348;010
47172>360:
77>27;4;
71>:90
:11898>422086
92:;4>:04:7:
5>3614
96>;:212
050230>02:
0:637>:
;219810;8>1;
:;;139>60:::855
3;:65;6:;>6439:1
97470:;2>1;2:9::7
76659>0
;900:>;3805;26
51;7;12>8111:95487
1560024>2;4645534
:32;98161>7828680
3549893>618
;6669722>389439:39
:6176:3;8>7;75:
2574>3587
953>125214
83182>8721189
7028>2546450285
42152>::
003810453>45903
5>8512:7
39:580>0361326:6
842;:7;65>8
;3:814>128
81;;;:5>:1
1;>05041334
4914:3>60
63:88>62
;160>64
109:860>5
9;789:>9;17147
:;18721>;079
4>716317539
98913>46502;612
10:569:44>0:71649
6>0:;921
8736>3812:;623
4;;;3>;12;310671
255:853>3696:
:171>80::313
;25>427
;0914>:26:475
75257>;:2
1;092>4585
4>167;:6
8>1
;09::313>118
@5928549;;01:82015;16:313;7448911679;;:23:41::533481642139:7970
@2705;302
@83468;1482304435223:2641:10512;993392462:926590837:912;33399:86;3219:7098631871;2;17543174:908972:
@15825756750:;3;;93223640:095375846:9178008:0;23055283:113
@:::3629442342405837;80821;16444226901;851;983113806442795039883:65274:499595:2072737124;28828446162
@4:6244;6:;11:900;8;64;8471;8755434:;09::0;3;073046:09:55;85;5370754;6296555608
@42850629:261798;8537;559323923;7:74256;7050;70:352;8;866:134823
@178;5;:59;44;7;59250:;7
@42519
@;28011;0;0374228;3:581871
@9:6:;9
@79601:311;
@85679405910;61104095;;3402;1723389;963634:6:;827040478872709:45:587331;8;5:255:612:8::349727;;55;40
@5;:451;2:90;;:72026:909137:2;:55:717801:12954875257
@:451;74;59874965283725061;349:0735860059119:3339364539458;19:145:1;259;;2895446;2;5681;804629
@;7739500421505964;9155686;0419618649;827230209188:26882677:188652225:400;07;980::286::0:52;
@39;9;53923118859857773596:;34257;89:20;
@7193161230124
@0203556349;1855;11438:9901:5363784;5198968;218679::68770059435;31292;5;6520;;66094:9;62
@3;8;4;252;;59;325
@182793375757;96783:28068:3;19
@9;54:6;6541;;00:;01921289030511346;64:0931865844052374384719024407:46;11427:242398:801980
@:046842700:516708:4356;36043891:195:17957;;05:420155;88
@859922770055;6;:52:3479582:7:74;410:47580;139023646;;38;96:;53298270258642956;81464710726964085935
@5:2;640;:083867387226082497500792108508708;02:42::5825
@72906273403598150181769287;4:4657891:99694;70401506
@:3
@72630:299:73044618:0164644097342529:0;;216512108928:81409814;9:9;0;4
@:65936;4:750841105:977045185009362010;93;27;:80322419457:1425153
@540913082;6;;955163:
@20962288675:;2462495:8405:5;5727731:61:01720
@:7424:546598285:2056674;9405:57103385804590896:3:318358
@9010981944595912:6:;4836483;2373793554611
@3:180423363:1778:9360:38304:69297750;837669;64785:6:19:822:8049;02;32629500340322:43;2054
@1832075;27:58759:1953;37731;8:92644631:690;974695721;6061251067
@3456570
@7;4272;12521;2;4886825759043741689870;8642789:45626881655:0
@66:63:2792155056::340950:;:71:;74168;878:9679;181
@4669690441781397397;9983597:587:22;13097547605
@3606322;7069965;9;3223;18897860345:774:6843:;010;42575785835679:0;9;789841635;79412371;0126727
@6:;69775303;6363785:7;
@;0458649168;47:9;;5:58803495;87;9489:56
@376725958231:45582:0;:382394078426;9306;22406992006;:1;0
@0857812313:19:8;60
@4:08::700:45202194349:2204:3;;171133;:330;82;370;34817:131924:95;082236626051860;:849;:147
@85518912480::7;3
@;99:09698424;:087544:7867746:0711831593629:5732286;757156:382:93:09424241499733822772048:890:689;
@51979:3:;;23;45637
@8898596
@532287;742331294512;819496468;:564870;2415;3
@0;85
@7239:433;;299815354:::715;2477:715409;3839905351813:3:081619538784;53266;368631794503994725155
@19;:154;40:4:165::;3500:8
@793114:0235;71;599:3796:4;:816297520144709672900;:7:;1835:36957
@2197613631593516794;2067::677142719164;9:87118;093;03:;58557;9962;;032:5:9:3593707775
@154603721114186324047:2252142::37618102253::342
@02274429:7106471201443:9:98212;6415197:83401882:08653562628;4;511;22:4:5836831
@399002760920629;78597100:7;:
@0111;65510626:807;74893:17086:::43;3958:961
@4140138143;9:;5923;673980979884010468480
@7::77;13:7258:595255;:7;217;10:92089;9028674
@1127;8:9723012215335;5509;04689;93;0;19:31;571:046
@7;3;82;40;54097108:;03:93377;15:4815249737;34:44677:257
@071;86917:6127852;9:95380::386
@81501496304460146
@556:023:0::::;38;9483708;704;698;36;9921;44237:24238978028552616109:1331873335:;;1;08
@03790232
@869675:059780064;73;834;:1349545324376303;08370767:622958199:2224:534078378;606178;2;97;:488:2:61
@9988934522587557;846;22;:1517:0980554136407;94680:234927:;09:06614355:31;:58
@3566;1:835:7985:63;::889;102684:53494690487140;27215:162264970;077101;457;;33:5:5:77;2960577:60
@11
@2010;0;6714962526;34;9;7;5;4:0245348473317171:
@;953062;57
@15332076836;67:;6090;:93379
@055528948838;1890:162262179426:6330:;96;5188289
@62195;3674292086488752;2472:7355666;91985;:8366;22;;8:109734:814687394077:334126715
@3:805;87;:45:;8;8867:9:22509695271244303188378275937:5:1:22907;70144
@879474;47;142867:7:1271960:;42802
@4128997:517;89;::15;230038739105261070201142;309861:15;81:;:;485:9
@6:4:;52549665474547259580945057:3;8743014:;22::8252767365;39527017;:243722
@50756471269838;012;67759190006
@9:024816:7
@6;3:3943332817:58:01;2257265150;8790457;
@9024:740;6320892263;678548223534550705;432632798158;04484::;4179:793407771
@980
@152
@5:816749;540831;826797:672487:04885679367;;3302360:3;826:66956156
@02070;;;66409:076;889:270755104569068;:796768862;6:9814802612:60;;:0207062:7
@492203;1;;515;789778673:279954;2514547:4;4543140041
@349:3678:6:307146;05362:;633242:0;83973;566
@17:77:52871;5438056086751:464:09:25542;5904287204000:;9;:5
@6424641;03:503:63914658
@4883:1549703;16449984;7798331415411236:42;850699375760770:2224;690007;512562;833792519120386;759851
@8:027:576821947;605:714;8908717747:;
@7:070699587555069;2;33:5:9167006;4232;07;70445:68490779:8:4428214457796:97214470144031:0;6826:7
@74:6245;3;47800984146489:8396;053205
@98281145555:798;951
@1:40213451321532:7;455246278;810;905;8
@411;2:29090469::4054465497926542:91477509643372;;;33910689162504368212220;6
@6079;2193173:814260948;9260256:3:501;5276:28:844241:5577;;:890;79;985333564420889262;59:
@;62;928039560106980180:264142315179:99;4683717523:43:2697711;6:017
@392732::4986062489:589:904;56;89828;4;2082;18194:726
@079576;:20680543;50185192
@010:906:55483:01:1:38:0
@;8515
@;90;823609395:;:;:395;750;154:5785:4;120251034:2613;0344292:8;54
@308820;;1;291842;7624;815694;506587;8
@:194;:07042::6970619:53;307:488695;0337772:
@07506624353;9395;:003;1;20:
@688781;2;167:26857189181494;26183173675
@8347527220:;6;4:0812323:7:
@15:;2571328940836;05;8:6319507:610976401:0:20::79:41:27674
@832282;7077167088::700;1;1851489723892399;37:74:969978262:3363466723154:8;6;62180;577851:88940327590
