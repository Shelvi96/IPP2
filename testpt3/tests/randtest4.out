0
0
0
0
0
0
0
91001010000
0
9110100000000
0
0
0
5764607523034234880
0
0
18430569501750722560
7
0
16807
459880526656507957
0
0
0
0
5764607523034234880
0
0
0
214358881
15085862483040141312
0
16117605711638167552
0
0
0
0
6701356245527298048
14095140933762809856
0
3104206659
17976804394182244960
0
1611853
0
13442
0
910011
0
15085862483040141312
0
0
10000000000000
0
0
6773244766835441664
0
41807320605137087
15092213262202175488
18252649210203602944
15085862483040141312
6629298651489370112
0
5058685793221587527
32768
0
0
0
0
0
0
14095140933762809856
0
3800665509557917
3120994541767753728
0
0
6110640388231970816
0
0
5104597550695543824
0
18252649210203602944
0
0
0
0
3261777232891331530
5764607523034234880
15092213262202175488
0
285311670611
0
4308165612584450826
0
4835855839594020864
4310658
13833710174285594624
0
0
0
13835058055282163712
98882502720768
31381059609
0
0
5705386105799243824
15092213262202175488
3832049362835804
5347198282864568438
348653455051946
0
0
